apps/ledger-live-desktop/jest.config.js

```diff
@@ -37,6 +37,7 @@ const commonConfig = {
     __SENTRY_URL__: null,
     __PRERELEASE__: "null",
     __CHANNEL__: "null",
+    Buffer: Uint8Array,
```

Comment:
this was necessary to fix issue where jsdom doesn't really support Buffer.

@noble/curve relies on Buffer being a subclass of UINT8Array. the implementation in JSDOM does not.

Reply:
resolved as of [10be86b](https://github.com/LedgerHQ/ledger-live/pull/7622/commits/10be86b6acf5ce8629a41aa6a74c5ec832e78883)